testutils/generators.go

```diff
@@ -32,16 +36,38 @@ func GenerateRecords(batchSize int) func() (record *exporter.Record, e error) {
 		return &exporter.Record{
 			RecordNumber: uint32(cnt),
 			Record: insrecord.Material{
-				ID:    gen.IDWithPulse(pn),
-				JetID: GenerateUniqueJetID(),
+				Polymorph: int32(randNum()),
+				Virtual: insrecord.Virtual{
+					Polymorph: int32(randNum()),
+					Union:     nil,
+					Signature: []byte{0, 1, 2},
+				},
+				ID:        gen.IDWithPulse(pn),
+				ObjectID:  gen.ID(),
+				JetID:     gen.JetID(),
+				Signature: []byte{0, 1, 2},
 			},
-			ShouldIterateFrom: nil,
+			ShouldIterateFrom: &pn,
+			Polymorph:         uint32(randNum()),
 		}, nil
 	}
 
 	return generateRecords
 }
 
+func GenerateRecordsList(count int) *[]exporter.Record {
```

Comment:
it will be better to rename `GenerateRecordsSilence`. so you have GenerateRecords which return an error and GenerateRecordsSilence wich not.